centos7/BUILD

```diff
@@ -75,6 +76,13 @@ container_image(
     ],
 )
 
+# Run the security check script to generate a metadata YAML file indicating
+# whether the centos 7 has critical security vulnerability fixes.
+security_check(
+    name = "metadata",
+    image = "gcr.io/asci-toolchain/centos7-xingao-test:latest",
```

Comment:
eventually we're going to want these to be gcr.io/gcp-runtimes/etcetc

Reply:
This needs to happen when we change the cloudbuild.yaml files.